LayoutTests/platform/mac-site-isolation/TestExpectations

```diff
@@ -372,6 +372,7 @@ accessibility/activation-of-input-field-inside-other-element.html [ Skip ]
 accessibility/add-children-pseudo-element.html [ Skip ]
 accessibility/ancestor-computation.html [ Skip ]
 accessibility/anonymous-render-block-in-continuation-causes-crash.html [ Skip ]
+accessibility/aria-controls-with-tabs.html [ Timeout ]
```

Comment:
Can we skip instead of mark as an expected timeout?  That'll help tests go faster, and that's what we do for other tests.